examples/mnist/conv_mnist.py

```diff
@@ -23,15 +26,15 @@
 
 parser = argparse.ArgumentParser()
 parser.add_argument("--seed", type=int, default=0)
-parser.add_argument("--n_train", type=int, default=60000)
+parser.add_argument("--n_epochs", type=int, default=1)
```

Comment:
Is this should be 60000 like in the `--n_train`?

Reply:
`--n_train` controls how many iterations. `--n_epochs` will control how many times through the dataset. Which for mnist (if using the standard train and test split) it will be 50k training steps for every epoch.

Reply:
60K, but I like this.